Comp2022/src/main/cpp/RobotContainer.cpp

```diff
@@ -184,22 +188,45 @@ void RobotContainer::ConfigureButtonBindings()
+
+    // frc2::JoystickButton m_climber0StowOp{ &m_operatorController, (int)frc::XboxController::Button::kA };
+    // frc2::JoystickButton m_climber1DeployOp{ &m_operatorController, (int)frc::XboxController::Button::kB };
+    // frc2::JoystickButton m_climber2ClimbL2Op{ &m_operatorController, (int)frc::XboxController::Button::kX };
+    // frc2::JoystickButton m_climber3RotateL3Op{ &m_operatorController, (int)frc::XboxController::Button::kY };
+    // frc2::JoystickButton m_climber4ExtendL3Op{ &m_operatorController, (int)frc::XboxController::Button::kLeftBumper };
+    // frc2::JoystickButton m_climber6ClimbToL3Op{ &m_operatorController, (int)frc::XboxController::Button::kStart };
+
+    frc2::POVButton m_climber0StowOp{ &m_operatorController, 180, 0 };
+    frc2::POVButton m_climber1DeployOp{ &m_operatorController, 0, 0 };
+    frc2::POVButton m_climber3RotateL3Op{ &m_operatorController, 270, 0 };
+    frc2::POVButton m_climber4ExtendL3Op{ &m_operatorController, 90, 0 };
+    frc2::JoystickButton m_climber5RotateInL3Op{ &m_operatorController, (int)frc::XboxController::Button::kStart };
+
+    frc2::JoystickButton m_climberClimbL2RotateL3{ &m_operatorController, (int)frc::XboxController::Button::kA };
+    frc2::JoystickButton m_climberClimbL3{ &m_operatorController, (int)frc::XboxController::Button::kY };
+
     frc2::JoystickButton m_intakingOp{ &m_operatorController, (int)frc::XboxController::Button::kLeftBumper };
     frc2::JoystickButton m_scoringPrimeOp{ &m_operatorController, (int)frc::XboxController::Button::kRightBumper };
-    frc2::JoystickButton m_shooterAimOnOp{ &m_operatorController, (int)frc::XboxController::Button::kStart };
+    // frc2::JoystickButton m_shooterAimOnOp{ &m_operatorController, (int)frc::XboxController::Button::kStart };
     frc2::JoystickButton m_shooterAimOffOp{ &m_operatorController, (int)frc::XboxController::Button::kBack };
-    frc2::JoystickButton m_climberMoveDistOp{ &m_operatorController, (int)frc::XboxController::Button::kY };
 
     // Operator - A, B, X, Y
-    m_inStowOp.WhenPressed(IntakeDeploy(false), true);
+    // m_inDeployOp.WhenPressed(IntakeDeploy(true), true);
+    // m_inStowOp.WhenPressed(IntakeDeploy(false), true);
     m_exhaustingOp.WhenPressed(ExhaustingAction(&m_intake, &m_floorConv, &m_vertConv), true);
     m_exhaustingOp.WhenReleased(ExhaustingStop(&m_intake, &m_floorConv, &m_vertConv), true);
-    // m_inDeployOp.WhenPressed(IntakeDeploy(true), true);
-    m_climberMoveDistOp.WhenPressed(ClimberMoveHeight(&m_climber), true);
+
+    m_climber0StowOp.WhenPressed(ClimberStow(&m_climber, &m_intake, &m_floorConv, &m_vertConv, &m_shooter), true);
+    m_climber1DeployOp.WhenPressed(ClimberDeploy(&m_climber), true);
+    m_climber3RotateL3Op.WhenPressed(ClimberRotateToL3(&m_climber), true);
+    m_climber4ExtendL3Op.WhenPressed(ClimberExtendToL3(&m_climber), true);
+    m_climber5RotateInL3Op.WhenPressed(ClimberRotateIntoL3(&m_climber), true);

```

Comment:
Wait why are there separate POV buttons for each climber stage but not for Climber2ClimbToL2? Is it just not necessary for testing?

Reply:
Climber2ClimbToL2 does technically the same thing as the 0StowHeight cmd (lower to 0.25 inches, set gatehook at default position), so it's not necessary to map it again for the sake of conserving buttons